src/components/common/DetailComment.vue

```diff
@@ -113,7 +116,6 @@ export default class DetailComment extends Vue {
   // Move the placeholder and input text from the edges.
   :deep() {
     .theme--light.v-input input, .theme--light.v-input textarea {
-      background-color: $gray1;
```

Comment:
The component had a weird behavior/color:
![weird color for detail comment](https://github.com/JazzarKarim/business-filings-ui/assets/122301442/1b055613-8b0a-4ed5-b445-aa1b533a1dfc)

I don't know if you can see it haha but the background is a little bit messed up (2 colors). This fixes that.